ceph: added rook-ceph-default service account (#7393)
rook/rook: Documentation/k8s-pre-reqs.md

After doing this for all service accounts all pods should be able to pull the image from your registry.

> ### Procedure for adding imagePullSecrets to service accounts if Helm Charts is used for Rook Installation :

Reviewer: I'm a fan of short titles
```suggestion
### Helm configuration
```

> In [Values.yaml](https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/values.yaml) populate imagePullSecrets, it will passed to all service accounts.

Reviewer: nit: add a blank line between the title and the paragraph. Also, remove extra lines before the next title.
```suggestion
In [Values.yaml](https://github.com/rook/rook/blob/master/cluster/charts/rook-ceph/values.yaml) populate `imagePullSecrets` to be set on all service accounts.
```

```yaml

> # imagePullSecrets option allow to pull docker images from private docker registry. Option will be passed to all service accounts.

Reviewer: Let's remove this comment since it's already explained.
